rules/macos/lateral_movement_remote_ssh_login_enabled.toml

```diff
@@ -36,6 +37,11 @@ framework = "MITRE ATT&CK"
 id = "T1021"
 name = "Remote Services"
 reference = "https://attack.mitre.org/techniques/T1021/"
+[[rule.threat.technique.subtechnique]]
+id = "T1021.004"
+name = "Remote Services: SSH"
```

Comment:
Will be correct when https://github.com/elastic/detection-rules/pull/2073 is merged